gateway/sds_gateway/api_methods/utils/metadata_schemas.py

```diff
@@ -387,9 +383,31 @@
 
 
 def get_mapping_by_capture_type(
-    capture_type: CaptureType,
+    capture_type: Any,
 ) -> dict[str, str | dict[str, Any]]:
     """Get the mapping for a given capture type."""
+    # Local import to avoid circular dependency
+    from sds_gateway.api_methods.models import CaptureType
+
+    # Initialize mapping if not already done
+    if not capture_index_mapping_by_type:
+        capture_index_mapping_by_type.update(
+            {
+                CaptureType.DigitalRF: drf_capture_index_mapping,
+                CaptureType.RadioHound: rh_capture_index_mapping,
+            }
+        )
+
+    # Handle string inputs by converting to enum
+    if isinstance(capture_type, str):
+        if capture_type == "drf":
+            capture_type = CaptureType.DigitalRF
+        elif capture_type == "rh":
+            capture_type = CaptureType.RadioHound
+        else:
+            msg = f"Invalid capture type string: {capture_type}"
+            log.error(msg)
+            raise ValueError(msg)
```

Comment:
not needed, just use `capture_type: CaptureType` directly as a dict key of `capture_index_mapping_by_type` below